content/release-notes/2022-10-04.md

```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Added social authentication for Hasura, enabling users to sign in using social networking services supported by Hasura.
+- Improved Neon project creation performance.
```

Comment:
Apologies. I missed this in my earlier review. Does this item have a label like "UI"?